tests/integration-tests/tests/schedulers/test_slurm/test_slurm_memory_based_scheduling/pcluster.config.mem-based-scheduling.yaml

```diff
@@ -27,7 +27,9 @@ Scheduling:
         - Name: ondemand1-i2
           Instances:
             - InstanceType: c5.xlarge
+            {% if "us-iso" not in region %}
```

Comment:
Here and below: why not replacing c5a.xlarge with c5d.xlarge for every region and avoid the conditional branch?